tools/fetch-external.sh

```diff
@@ -4,7 +4,7 @@
 # get latest CLI docs
 cd pages/reference/ && {
   curl -O -L https://github.com/balena-io/balena-cli/raw/master/doc/cli.markdown
-  mv cli.markdown cli.md
+  mv cli.markdown balena-cli.md
```

Comment:
In case this needs clarification, the external docs use a regex and so I needed to make this less generic to avoid conflicts with the masterclass CLI. I added in all the relevant redirects.